src/source/tile_bounds.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const LngLatBounds = require('../geo/lng_lat_bounds');
+const clamp = require('../util/util').clamp;
+
+class TileBounds {
+    constructor(bounds, minzoom, maxzoom) {
+        this.bounds = LngLatBounds.convert(bounds);
+        this.minzoom = minzoom || 0;
+        this.maxzoom = maxzoom || 24;
+    }
+
+    contains(coord) {
+        const level = [
+        [Math.floor(this.lngX(this.bounds.getWest(), coord.z)), Math.floor(this.latY(this.bounds.getNorth(), coord.z))],
+        [Math.ceil(this.lngX(this.bounds.getEast(), coord.z)), Math.ceil(this.latY(this.bounds.getSouth(), coord.z))]
+        ];
```

Comment:
nit: for readability, instead of an array, could this be four individual `const`s `minX`, `minY`, etc.?